src/data/team.json

```diff
@@ -5,62 +5,62 @@
     "role": "Team lead",
     "photoUrl": "olga-tiutiunnyk",
     "githubUrl": "https://github.com/izachok",
-    "text": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum vitae saepe odio consectetur, optio doloribus, soluta reiciendis officiis consequatur qui molestiae tempore expedita."
+    "text": "Olga is our Teamlead. She prepared structure of project, wrote documents about project interfaces and made many reviews. She implemented footer and this modal about team with slider, wrote functions for working with API, refactored project in order to connect all parts together. Add to library buttons also were implemented by her. "
   },
   {
     "fullName": "Hanna Hryhorieva",
     "city": "Ukraine, Kharkiv",
     "role": "SCRUM master",
     "photoUrl": "hanna-hryhorieva",
     "githubUrl": "https://github.com/HannaHryhorieva",
-    "text": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum vitae saepe odio consectetur, optio doloribus, soluta reiciendis officiis consequatur qui molestiae tempore expedita."
+    "text": "Hanna is our Scrum master. Thanks to her, the evening standup turned into convenient and useful pastime for our team. She wrote functional for rendering cards and their styles and added notifications about errors."
```

Comment:
Hannah is our Scrum Master. Thanks to her,  Daily standup meetings were held smoothly,  efficiently and at a time convenient for all the team members.   She makes functionality and styles for rendering movie cards as well as for errors notification.